frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/inventory-card.tsx

```diff
@@ -65,6 +71,10 @@ const InventoryCard: React.FC<DashboardItemProps> = ({
   const pvsLoadError = _.get(resources.pvs, 'loadError');
   const pvsData = _.get(resources.pvs, 'data', []) as K8sResourceKind[];
 
+  const scData = _.get(resources.sc, 'data', []) as K8sResourceKind[];
+  const filteredCephSc = getCephSC(scData);
```

Comment:
```suggestion
  const filteredCephSC = getCephSC(scData);
```
Its a nit but I think we should keep the SC naming convention consistent throughout.